de.marw.cdt.cmake.core/src/main/java/de/marw/cdt/cmake/core/internal/BuildscriptGenerator.java

```diff
@@ -377,6 +445,54 @@ private MultiStatus invokeCMake(IPath srcDir, IPath buildDir, IConsole console)
       return new MultiStatus(CdtPlugin.PLUGIN_ID, IStatus.ERROR, errMsg, null);
     }
   }
+  
+  private void tryAddMakeTarget(
+      IMakeTargetManager tm,
+      String buildLocation,
+      String targetBuilder,
+      String targetName) {
+    try {
+      final IMakeTarget target = tm.createTarget(project, targetName, targetBuilder);
+      target.setStopOnError(false);
+      target.setRunAllBuilders(false);
+      target.setUseDefaultBuildCmd(true);
+      target.setBuildAttribute(IMakeCommonBuildInfo.BUILD_COMMAND, "make");
+      target.setBuildAttribute(IMakeTarget.BUILD_LOCATION, buildLocation);
+      target.setBuildAttribute(IMakeTarget.BUILD_ARGUMENTS, "");
+      target.setBuildAttribute(IMakeTarget.BUILD_TARGET, targetName);
+      tm.addTarget(project, target);
+    } catch (CoreException e) {
+      log.log(new Status(IStatus.ERROR, CdtPlugin.PLUGIN_ID,
```

Comment:
Most users do not read the Eclipse log. Please find a way to report is as markers or a Status